src/appliance_setup/avs/avsarconboarder/collector/_DataCollector.py

```diff
@@ -24,8 +24,8 @@ def _build_customer_cloud_retriever(self, retrieval_factory):
 
     def collect_data(self, *args):
         customer_res: CustomerResource = args[0]
-        cloud_details, customer_credentials = self._collect_customer_cloud_data(customer_res)
-        vsphere_resource = self._vsphere_resource_retriever.retrieve_data(None)
+        cloud_details, customer_credentials = self._collect_customer_cloud_data(customer_res)        
```

Comment:
NIT : Trailing whitespaces

Reply:
Was this an older bug?
The _collect_customer_cloud_data always seemed to be expecting customer_res, right?

Reply:
removed trailing spaces

Reply:
I didn't get this part. Previously also customer_res was being passed to _collect_customer_cloud_data()

Reply:
My bad, makes sense